### PR TITLE
Fix Location-Master-bug

### PR DESCRIPTION
**Pull request by MonishaSivanathan**
- update labtest master
- splitted function according to the patient, admin module

### CODE REVIEW

File: routes/master_routers.py [hunk 1]
```diff
@@ -13,18 +13,24 @@
...
+    return MasterClient.get_service_type()
+
 @router.get("/master/{table_name}")
 @permission_required("MASTER", "READ")
 async def get_master_value(table_name: str, request: Request):
     logger.info(f"master table {table_name}")
     return MasterClient.get_master_data(table_name)
 
-@router.get("/master/{table_name}/filtered")
+@router.get("/master/{table_name}/admin_filtered")
```
Comment (tamsud) [inline routes/master_routers.py:29]: rename this one?

File: routes/master_routers.py [hunk 2]
```diff
@@ -33,7 +39,20 @@ async def get_master_data(
 ):
     logger.info(f"master table {table_name}")
     return MasterClient.fetch_master_data(
-        table_name, all_records, code, display, service_type, page, page_size
+        table_name, code, display, service_type, page, page_size
+    )
+
+@router.get("/master/{table_name}/patient_filtered")
```
Comment (tamsud) [inline routes/master_routers.py:45]: rename this one?

**Review by tamsud — CHANGES_REQUESTED** [review]

File: controller/location_controller.py [hunk 3]
```diff
@@ -127,20 +120,32 @@ def get_all_location(active: bool, location: str = "", city_state: str = "", pag
...
             return response
         except Exception as e:
             return JSONResponse(
                 content={"error": "Unable to retrieve location", "details": str(e)},
                 status_code=status.HTTP_400_BAD_REQUEST
             )
         
+    @staticmethod
+    def get_all_location(city: str, state: str,
+                        location: str, page: int, page_size: int):
```
[the remark this answers is not in the document]
Comment (MonishaSivanathan) [reply controller/location_controller.py]: no tamil, i have changed

File: routes/master_routers.py (hunk 1 above)
Comment (MonishaSivanathan) [reply routes/master_routers.py]: update the name to search

File: routes/master_routers.py (hunk 2 above)
Comment (MonishaSivanathan) [reply routes/master_routers.py]: update the name to search

File: routes/master_routers.py [hunk 4]
```diff
@@ -13,18 +13,24 @@
...
+    return MasterClient.get_service_type()
+
 @router.get("/master/{table_name}")
 @permission_required("MASTER", "READ")
 async def get_master_value(table_name: str, request: Request):
     logger.info(f"master table {table_name}")
     return MasterClient.get_master_data(table_name)
 
-@router.get("/master/{table_name}/filtered")
+@router.get("/master/{table_name}/admin_search")
```
Comment (tamsud) [inline routes/master_routers.py:29]: can you rename the url path to  asmin/master/{table_name}/filtered

File: routes/master_routers.py [hunk 5]
```diff
@@ -33,7 +39,20 @@ async def get_master_data(
 ):
     logger.info(f"master table {table_name}")
     return MasterClient.fetch_master_data(
-        table_name, all_records, code, display, service_type, page, page_size
+        table_name, code, display, service_type, page, page_size
+    )
+
+@router.get("/master/{table_name}/patient_search")
```
Comment (tamsud) [inline routes/master_routers.py:45]: can you rename the url path to  patient/master/{table_name}/filtered

**Review by tamsud — CHANGES_REQUESTED** [review]

File: routes/master_routers.py (hunk 4 above)
Comment (MonishaSivanathan) [reply routes/master_routers.py]: changed mean

File: routes/master_routers.py (hunk 5 above)
Comment (MonishaSivanathan) [reply routes/master_routers.py]: changed to the Endpoint

**Pull request merged**